tests/virtualization/xen/hotplugging.pm

```diff
@@ -22,37 +22,71 @@ use version_utils;
 sub run_test {
     my ($self) = @_;
     my $hypervisor = get_var('HYPERVISOR') // '127.0.0.1';
+    my ($sles_running_version, $sles_running_sp) = get_sles_release;
+
+    my %mac = ();
+    record_info "Virtual network", "Adding virtual network interface";
+    foreach my $guest (keys %xen::guests) {
+        $mac{$guest} = '00:16:3f:32:' . (int(rand(89)) + 10) . ':' . (int(rand(89)) + 10);
+        unless ($guest =~ m/hvm/i && is_sle('<=12-SP2') && check_var("XEN", "1")) {
+            my $persistent_config_option = '';
+            my $interface_model_option   = '';
+            if (get_var('VIRT_AUTOTEST')) {
```

Comment:
```suggestion
```

Reply:
Same thought as above.

Reply:
In here, I the `if (get_var('VIRT_AUTOTEST')) {` is not needed. Please remove it.

Reply:
It is needed here unless you confirm that this part will also be used by some other people.

Reply:
I did not remove it, but moved it into respective kvm and xen branches. I think this way is less layer of indention and looks nicer.

Reply:
OK. Maybe keep this condition.